.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+@BalzaniEdoardo @billbrod @ahwillia @gviejo
```

Comment:
Confused about this codeowners file: my understanding is that these files should be structured like `path @user`, where `user` is then the owner for every file at `path` (as in [the docs](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners)). Thus, I was expected this to be `* @BalzaniEdoardo`, meaning Edoardo needs to approve every PR to this repo. I think there's something weird here where it thinks Edoardo is the path. Also, in general, I thought we had agreed to have one owner per repo (else, we'd all have to review every PR), potentially making different people "subowners" (e.g., making me the owner of `ci.yml` or `.readthedocs.yml` or Guillaume the owner of `tox.ini`)

Reply:
with this syntax, I am setting up everyone as code owner. I suggested in the comments that we can make it more specific later but since we did not discuss the specifics yet I went for this temporary solution. If you prefer I can set myself as the only owner for now or add you guys as specific script owner as you are suggesting

Reply:
edited, for now I set myself as codeowner, we can be more specific  once we agree on who does what on this repo